src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DataMaskingTests.cs

```diff
@@ -25,12 +25,13 @@ public class DataMaskingTests : SqlTestsBase
     {
         protected override void SetupManagementClients(RestTestFramework.MockContext context)
         {
-            var sqlCSMClient = GetSqlClient();
+            var sqlClient = GetSqlClient(context);
```

Comment:
It's not overridden in almost every test class. There are 20 other test classes that you aren't seeing in this review because they are unchanged.

The overrides are not the same as the base method. Some use StorageClient, some use StorageClientV2. Some use ResourcesClient, some use MockDeploymentClientFactory.GetResourceClient(GetResourcesClient()). The code looks similar but it's not exactly the same.

I could make some minor refactorings here, but I don't see the point. I'm making an infrastructure change in order to unblock our organization from fully adopting Swagger, and I don't think it makes sense to hold this up on minor code cleanups that can be done in the many files that I'm touching.